crates/oxvg_optimiser/src/jobs/convert_colors.rs

```diff
@@ -86,34 +87,46 @@ impl<E: Element> Visitor<E> for ConvertColors {
         let is_masked = element.local_name() == mask_localname
             || element.closest_local(mask_localname).is_some();
 
+        let mut method = self.method.clone().unwrap_or_default();
+        if is_masked && matches!(method, Method::CurrentColor) {
+            method = Method::Lightning;
+        }
         for mut attr in element.attributes().into_iter_mut() {
-            let is_style = attr.local_name().as_ref() == "style";
-            let style = if is_style {
-                attr.value().to_string()
-            } else {
-                format!("{}:{}", attr.local_name(), attr.value())
-            };
-            let style = StyleAttribute::parse(&style, ParserOptions::default());
-            let mut style = match style {
-                Ok(result) => result,
-                Err(e) => {
-                    log::debug!("failed to convert {}: {e}", attr.formatter());
-                    continue;
-                }
-            };
+            if attr.local_name().as_ref() == "style" {
+                let style = attr.value().to_string();
+                let style = StyleAttribute::parse(&style, ParserOptions::default());
+                let mut style = match style {
+                    Ok(result) => result,
+                    Err(e) => {
+                        log::debug!("failed to convert {}: {e}", attr.formatter());
+                        continue;
+                    }
+                };
 
-            let mut method = self.method.clone().unwrap_or_default();
-            if is_masked && matches!(method, Method::CurrentColor) {
-                method = Method::Lightning;
-            }
-            method.convert_style(&mut style);
-            let mut minified_style = method.to_css(&style).unwrap();
-            if !is_style {
-                if let Some((_, value)) = minified_style.split_once(':') {
-                    minified_style = value.trim_start().to_string();
+                method.convert_style(&mut style);
+                let minified_style = method.to_css(&style).unwrap();
+                attr.set_value(minified_style.into());
+            } else {
+                let minified_value = if let Some(mut presentation) = attr.presentation() {
+                    if method.convert_presentation(&mut presentation) {
+                        Some(
+                            presentation
+                                .value_to_css_string(PrinterOptions {
+                                    minify: true,
+                                    ..Default::default()
+                                })
+                                .unwrap(),
```

Comment:
```suggestion
                                .ok(),
```
Don't need to panic? We can instead leave the value as is